### PR TITLE
Removes felinids*

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
***and wraps them into humans. Existing felinids savefiles will be turned into humans with cat ears and a cat tail.**
image

note- doesn't remove purrbation. **you are not safe.**

## Why It's Good For The Game
Let's face it- felinids were just humans with minor tweaks (that were even so minor as to be disabled by some people resulting in literally just humans). This wraps them into being modified humans, which makes a lot more sense than having them be their own species with all the baggage that entails lore and otherwise. Also allows us to add more variations of tails and whatnot without having to worry about the species being explicitly feline!

## Changelog
:cl:
del: Felinid mutation toxin
del: Weird flavouring of tackles were removed. (E.g. pounce vs. tackle)
del: Felinids
add: Humans can now select ears and tails
fix: All species are now able to wag tails if they get them.
/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/purrbation.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	purrbation_remove(H, silent)
+	. = FALSE
+
+/proc/purrbation_apply(mob/living/carbon/human/H, silent = FALSE)
+	if(!ishuman(H))
+		return
+	var/obj/item/organ/ears/cat/kitty_ears = new
+	var/obj/item/organ/tail/cat/kitty_tail = new
+	kitty_ears.Insert(H, TRUE, FALSE) //Gives nonhumans cat tail and ears
+	kitty_tail.Insert(H, TRUE, FALSE)
```
Comment (Fikou) [inline code/modules/admin/verbs/purrbation.dm]: use named args if possible

File: code/modules/admin/verbs/purrbation.dm [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	if(!silent)
+		to_chat(H, "<span class='boldnotice'>Something is nya~t right.</span>")
+		playsound(get_turf(H), 'sound/effects/meow1.ogg', 50, TRUE, -1)
```
Comment (Fikou) [inline code/modules/admin/verbs/purrbation.dm]: get turf is not needed here

File: code/modules/admin/verbs/purrbation.dm [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/proc/mass_purrbation()
+	for(var/M in GLOB.mob_list)
```
Comment (Fikou) [inline code/modules/admin/verbs/purrbation.dm]: use descript variable names for all these please
also i think we have better global lists for that, like living mobs, or players.

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/purrbation.dm (hunk 3 above)
Comment (MarkSuckerberg) [reply code/modules/admin/verbs/purrbation.dm]: oh yeah, I just copied it wholesale from the old felinid file, I can get them fixed up when I'm home

File: code/modules/admin/verbs/purrbation.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/admin/verbs/purrbation.dm]: also were sure this isnt supposed to be like, SEND_SOUND?

**Comment by MarkSuckerberg** [comment]
Everything resolved, Purrbation optimisations at 500%

**Comment by triplezeta** [comment]
oh no you dont

**Comment by MarkSuckerberg** [comment]
Merge conflicts fixed

**Review by Fikou — APPROVED** [review]

**Pull request merged**